v2/emailpassword/migration/about.mdx

```diff
@@ -9,42 +9,27 @@ hide_title: true
 
 # About
 
-In this guide we will be going through the process of migrating users from an external Authentication provider to SuperTokens.
+This guide explains how to migrate users from an external authentication provider to SuperTokens.
 
-There are 3 steps to user migration:
+Migration involves two main steps:
 
-- Account Migration
-  - User Creation
-  - UserId Mapping
-  - Mark email as verified
-- User Data Migration
-- Session Migration
+Account Migration
+Session Migration
 
 ## Step 1) Account Migration:
 
-### User Creation
-- Our first step involves creating a SuperTokens user by importing their account credentials from the previous authentication provider.
-- You can learn more about how to implement these changes in the [User Creation](./account-creation/user-creation) section. 
+In the first step, use the Bulk User Import API to create a SuperTokens user for each user in your existing authentication provider. This API allows you to:
 
-### User ID Mapping
-- If you have stored information against existing userIds in your application table, you can use UserId Mapping to map the existing userId to the user's SuperTokens userId.
-- Once the userIds are mapped you can use the existing userId to interact with all of SuperTokens APIs.
-- You can learn more about how to implement these changes in the [User Id Mapping](./account-creation/user-id-mapping) section.
+- Create a SuperTokens user.
+- Map the existing userId to the SuperTokens userId.
+- Mark the email as verified (if applicable).
+- Add roles, metadata, and MFA devices (if applicable).
+- Perform account linking (if applicable).
```

Comment:
You should probably say linking multiple login methods for same user. Not sure if people understand what account linking would mean.

Reply:
We have used the term "account linking" in our docs before too.